dev_sandbox/main_linux.cpp

```diff
@@ -12,7 +12,6 @@
 #include <X11/X.h>
 
 #include <vulkan_wrapper.h>
-#include <vulkan/vulkan_xlib.h>
```

Comment:
1. Sorted includes cause build failure.
2. This include here is not needed, as it is included inside vulkan_wrapper.h